cfgov/v1/static/js/admin/html_editor.js

```diff
@@ -0,0 +1,79 @@
+(function() {
+  (function($) {
+    return $.widget('IKS.editHtmlButton', {
+      options: {
+        uuid: '',
+        editable: null
+      },
+      populateToolbar: function(toolbar) {
+        if (window.location.href.indexOf('ask_cfpb') > -1) {
```

Comment:
I see, this is a nice way to limit, although it'd be really nice if this was somehow a little more straightforward or could be configured via some kind of variable passed into the JS.